example/test/get_dom_node_test.dart

```diff
@@ -41,7 +42,7 @@ class SimpleComponent extends react.Component {
     react.div({}, [
       react.span({"ref": "refToSpan"}, "Test"),
       react.span({}, counter),
-      react.button({"onClick": (_) => (getDOMNode() as HtmlElement).children.first.text = (++counter).toString()},"Increase counter"),
+      react.button({"onClick": (_) => (react_dom.findDOMNode(this) as HtmlElement).children.first.text = (++counter).toString()},"Increase counter"),
```

Comment:
Why'd we drop `getDOMNode()`? It was pretty convenient and we use it in a bunch of places.